Dockerfile

```diff
@@ -25,4 +26,9 @@ WORKDIR /code
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 # build assets
-RUN npm ci && npm run build
+RUN cd taccsite_custom && npm ci
+RUN npm run build --project=$PROJECT_NAME\
+    # copy base core cms stylesheets to legacy path
```

Comment:
**Yes.**

On `main` branch, the `python manage.py collectstatic` command added built stylesheets into `taccsite_cms/static/site_cms/css/build`. Many paths in HTML files expect the stylesheets to exist there.

<details>
<summary><strong>Why Not Just Update the Paths?</strong></summary>

<details>
<summary>Limit Testing</summary>

I want to minimize the things I need to test. Testing now is simpler:
- The build fails _or not_.
- **All** styles load _or not_.

</details>

<details>
<summary>Limit Scope</summary>

I want to avoid PRs for two more repos (Portal and Docs have paths), because I thought this change was already quite large in scope and I am not sure whether these are **all** the paths.

</details>

| Docs | Portal | CMS |
| - | - | - |
| ![FP-1497 Static Dir - Docs](https://user-images.githubusercontent.com/62723358/155603077-73716e48-4187-407c-913a-c96d65c11eae.png) | ![FP-1497 Static Dir - Portal](https://user-images.githubusercontent.com/62723358/155603089-fe39001c-d918-42e1-a688-245ed5441d2a.png) | ![FP-1497 Static Dir - CMS](https://user-images.githubusercontent.com/62723358/155603093-4f990c05-08a3-4919-95af-df03d6183eb5.png) |

</details>

I intended to create a ticket to update the path, but I **am** willing to do it in this PR.

Reply:
This change is absent in upcoming PR.